### PR TITLE
Cache maps dialog

### PR DESCRIPTION
**Pull request by apbuteau**
Button that opens a dialog so the user can enter a lat/long to change the map viewscape.  This forces mapbox to download the desired tiles which are stored in the browser cache.
[Screen Recording 2020-11-30 at 11.35.51 PM.mov.zip](https://github.com/facebookexperimental/rf-coverage-maps/files/5620932/Screen.Recording.2020-11-30.at.11.35.51.PM.mov.zip)

### CODE REVIEW

File: app/src/components/CacheMapsDialog.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import PropTypes from 'prop-types';
+import TextField from '@material-ui/core/TextField';
+
+function CacheMapsDialog(props) {
```
Comment (tcirstea) [inline app/src/components/CacheMapsDialog.js:24]: FlowTypes for props?
Edit: I see you have the old proptypes

File: app/src/components/CacheMapsDialog.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+                </Button>
+                <Button onClick={handleDownload} color="primary">
+                    Download
+                </Button>
+            </DialogActions>
+        </Dialog>
+    );
+}
+
+CacheMapsDialog.propTypes = {
```
Comment (tcirstea) [inline app/src/components/CacheMapsDialog.js:86]: Why not use flow?

File: app/src/screens/mapscreen.js [hunk 3]
```diff
@@ -218,6 +224,37 @@ function MapScreen(): React.Node {
...
+
+/*
+  const [view, setView] = useState<ViewState>({
+    latitude: 37.483175,
+    longitude: -122.150084,
+    zoom: 17,
+    bearing: 0,
+    pitch: 45,
+  });
+*/
```
Comment (tcirstea) [inline app/src/screens/mapscreen.js:253]: ?

File: app/src/components/CacheMapsDialog.js (hunk 2 above)
Comment (apbuteau) [reply app/src/components/CacheMapsDialog.js:86]: Will do.

File: app/src/screens/mapscreen.js (hunk 3 above)
Comment (apbuteau) [reply app/src/screens/mapscreen.js:253]: doh

**Comment by apbuteau** [comment]
I'll fix everything in a new commit :-(

**Pull request merged**